pkg/operator/ceph/cluster/cluster_test.go

```diff
@@ -292,3 +279,72 @@ func TestTelemetry(t *testing.T) {
 		c.reportTelemetry()
 	})
 }
+
+// func TestAssimilateConf(t *testing.T) {
```

Comment:
I work on a new unit test since we remove the file so need to work how we can collect the configuration in mon store